test/ranges.jl

```diff
@@ -561,6 +570,10 @@ replstr(x) = stringmime("text/plain", x)
 # to cover the designated screen size.
 @test replstr(0:10^9) == "1000000001-element UnitRange{$Int}:\n 0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,…,999999998,999999999,1000000000"
 
+@test sprint(io -> show(io,UnitRange(1,2))) == "1:2"
```

Comment:
FYI I think you can do this as

    @test sprint(show,UnitRange(1,2)) == "1:2"